scripts/aws/setupdb.sh

```diff
@@ -40,12 +45,26 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 # Run migrations
 envdir /etc/mmw.d/env /opt/app/manage.py migrate
 
-if [ "$load_data" = "true" ] ; then
-	# Fetch boundary layer sql files
-	FILE_HOST="https://s3.amazonaws.com/data.mmw.azavea.com"
-	FILES=("boundary_huc12.sql.gz" "boundary_huc10.sql.gz" "boundary_huc08.sql.gz")
+function download_and_load {
+    FILE_HOST="https://s3.amazonaws.com/data.mmw.azavea.com"
```

Comment:
Might make sense to pull this variable out as a global instead of reassigning it every time this function is called.